client/src/pages/ResetPasswordRequest.jsx

```diff
@@ -0,0 +1,123 @@
+import { Formik, Form, Field } from 'formik';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { useState } from 'react';
+import { usePageError } from '../hooks/usePageError.js';
+import { authService } from '../services/authService.js';
+
+function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+export const ResetPasswordRequest = () => {
+  const [emailSent, setEmailSent] = useState(false);
+  const [error, setError] = usePageError('');
+
+  if (emailSent) {
+    return (
+      <section className="">
+        <h1 className="title">Check your email</h1>
+        <p>We have sent you following instructions</p>
+      </section>
+    );
+  }
+
+  return (
+    <>
+      <Formik
+        initialValues={{
+          email: '',
+        }}
+        validateOnMount={true}
+        onSubmit={({ email }, formikHelpers) => {
+          formikHelpers.setSubmitting(true);
+
+          authService
+            .resetPasswordRequest({ email })
+            .then(() => {
+              setEmailSent(true);
+            })
+            .catch((error) => {
+              if (error.message) {
+                setError(error.message);
+              }
+
+              if (!error.response?.data) {
+                return;
+              }
+
+              const { errors, message } = error.response.data;
+
+              formikHelpers.setFieldError('email', errors?.email);
+
+              if (message) {
+                setError(message);
+              }
+            })
+            .finally(formikHelpers.setSubmitting(false));
```

Comment:
The `finally` block should contain a function call, not the function itself. Wrap `formikHelpers.setSubmitting(false)` in an arrow function: `() => formikHelpers.setSubmitting(false)`.